src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -266,7 +267,30 @@ public abstract class AbstractAnalyzeMojo
     // defaultValue value on @Parameter - not work with Maven 3.2.5
     // When is set defaultValue always win, and there is no possibility to override by plugin configuration.
     @Parameter
-    private List<String> ignoredPackagings = Arrays.asList( "pom", "ear" );
+    private final List<String> ignoredPackagings = Arrays.asList( "pom", "ear" );
+
+
+    /**
+     * List of dependencies to be included. The result of this list is then applied to the ignore parameters.
+     *
+     * The filter syntax is:
+     *
+     * <pre>
+     * [groupId]:[artifactId]:[type]:[version]
+     * </pre>
+     *
+     * where each pattern segment is optional and supports full and partial <code>*</code> wildcards. An empty pattern
+     * segment is treated as an implicit wildcard. *
+     * <p>
+     * For example, <code>org.apache.*</code> will match all artifacts whose group id starts with
+     * <code>org.apache.</code>, and <code>:maven-dependency-plugin</code> will result in the analysis being applied
```

Comment:
This is confusing. Try using an example of an unrelated library instead like JodaTime.

Reply:
Done by copying the example Javadoc above. Decided the purpose of this section of the Javadoc is to explain the Maven co-ordinate and wildcard combination rather than the specific behaviour of the parameter.